src/common/options/rgw.yaml.in

```diff
@@ -3240,6 +3240,20 @@ options:
   - rados
   - dbstore
   - motr
+- name: dbstore_db_path
+  type: str
+  level: advanced
+  desc: experimental Option to set db backend store file path
+  default: /var/run/ceph
+  services:
+  - rgw
+- name: dbstore_db_name_prefix
+  type: str
+  level: advanced
+  desc: experimental Option to set db backend store file name prefix
```

Comment:
ditto

Reply:
(at least, I don't see what's experimental about this, even if we got rid of it at some point)

Reply:
done.